src/hexo/themes/sonarwhal/source/js/scanner-submit.js

```diff
@@ -434,7 +401,7 @@
                 hideQueueMessage();
 
                 if (isError) {
-                    updateScanFailUI(response);
+                    updateScanFailUI();
```

Comment:
If `isError` is true, `setTimeout(queryAndUpdate, 5000);` will never be reached and no more queries will be sent, does this need to be changed so that the page keeps updating in case more errors/warnings coming in? Also, what about the rules in the worker that has failed, we should show `scan failed` messages for those rules, right?

Reply:
>If isError is true, setTimeout(queryAndUpdate, 5000); will never be reached and no more queries will be sent, does this need to be changed so that the page keeps updating in case more errors/warnings coming in?

The status `error` is send at the end of the scan, so we don't need more updates after the error.

> Also, what about the rules in the worker that has failed, we should show scan failed messages for those rules, right?

Yes, the scanner set the status to error in each rule configured in the worker and at the same time add a message, so we don't need to change anything in the site :)

Reply:
> Yes, the scanner set the status to error in each rule configured in the worker and at the same time it adds a message, so we don't need to change anything in the site :)

:+1: